src/strategies/index.ts

```diff
@@ -419,6 +419,7 @@ const strategies = {
   'erc1155-balance-of-cv': erc1155BalanceOfCv,
   'prepo-vesting': prepoVesting,
   multichain,
+  'multichain-max': multichainMax,
```

Comment:
Thanks, @sirpy Make sure it is changed here too :) also make sure to remove any changes not related to this strategy. example change in `src/strategies/math/index.ts` make sure to revert, everything else looks good

Reply:
done

Reply:
hmm still issues looking at it

Reply:
all done